### DISCUSSION TITLE
Feature request: mouse cursor scale

### DISCUSSION THREAD

**Discussion by Hezkore** [OpenTTD/OpenTTD#11736] [Features]
I run the game at double interface scale with a TTF font.
Most of the game looks great, but the mouse cursor looks like a giant hideous monster.
I'd really like an option _(just a checkbox)_ for the mouse cursor to NOT be scaled with the rest of the interface.

I think everyone gets the idea here.
But I've been fiddling around with the code myself, and this is before and after.

Before/Scaled Cursor:
![big](https://user-images.githubusercontent.com/1727541/114287924-7e88ba00-9a6b-11eb-8e41-de25dc216b9e.png)

After/Unscaled Cursor:
![small](https://user-images.githubusercontent.com/1727541/114287929-821c4100-9a6b-11eb-8b6a-029e2a3fee8f.png)

**Comment by LordAro** [comment]
Not a bad idea - I think a setting with "GUI size", "1x", "2x", "4x" options would work quite well (defaulting to GUI size)

**Comment by PeterN** [comment]
1) You should see the cursor from the original graphics!
2) There are special cursors that are built up from engine parts (drag & drop) which would need attention.
3) Less is more.

**Comment by Hezkore** [comment]
I'd be happy with just a "Scale mouse pointer" checkbox, that scales the mouse pointer with the interface scale.

**Comment by nielsmh** [comment]
There is also a (separate) concern about having a high-contrast mouse cursor, possibly as a forced override of the baseset.
